en/01_Agenda/v1/04_Snippets/01_AgendaList.md

```diff
@@ -27,6 +27,7 @@ locations | Comma separated list of aliases of locations to filter the displayed
 locationTpl | Name of a chunk that contains the template for the location of an event. | tplAgendaEventLocation
 offset | An offset of locations returned by the criteria to skip. | 0
 outputSeparator | An optional string to separate each tpl instance. | -
+resourceTpl | Name of a chunk that contains the template for the linked resource of an event. | -
```

Comment:
If this was added in 1.2.1, might be worth noting that in the description? Just to avoid people on 1.2.0 or older trying to use it unsuccessfully.

Reply:
It was a hidden property, that could be filled before. Detected during the rewrite for MODX3.